### DISCUSSION TITLE
Complete Tutorial Video for OpenAI's Whisper Model for Windows Users

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [openai/whisper#106] [Show and tell]
Thank you so much for this amazing release Open AI. 

This is the first time ever I am able to utilize an AI public release and it actually works.

I was in need to speech to text services and best working one was Google's. However it was too expensive for me.

Now with Open AI Whisper I have even a better one for free.

By the way I also have uploaded multilingual large model generated subtitles and medium.en model subtitles as well without touching them. So you can also analyze their performance.

Choose subtitles from the settings panel of the video as below:

multilingual large model > _English (Canada) - OpenAI > Whisper > Large Multilingual Model > Automatic Subtitle > Raw_

medium.en model > _English (Ireland) - OpenAI > Whisper > medium.en Model > Automatic Subtitle > Raw_

Here my video : How to do Free Speech-to-Text Transcription Better Than Google Premium API with OpenAI Whisper Model

https://youtu.be/msj3wuYf3d8

_In this video : I will show you how to install the necessary Python code and the dependent libraries. I will show you how to download a video from YouTube with YT-DLP, how to cut certain parts of the video with LosslessCut, and how to extract the audio of a video with FFMPEG. I will show you how to do a transcription of a video or a sound. I will show you how to generate subtitles for any video. Finally, I will show you how to generate translated transcription and subtitles of any language video._

**Comment by FurkanGozukara** [comment]
After my RTX 3060 card has arrived I made this performance testing video : https://www.youtube.com/watch?v=q8Q8CCDdSKo

Moreover, after my card has arrived, I have done translation tests and  I have noticed that `--compression_ratio_threshold` helps tremendously in translation when translation fails and generates repeating output

**Reply by dgoryeo** [reply]
Have you come to find a magic number for --compression_ratio_threshold value? I am having similar problem with repeating output / and halucination output.

**Reply by FurkanGozukara** [reply]
No magic number. Also translation is still very inferior.

**Reply by andresgeorge** [reply]
you kind of lost me in the video how  do you translate after downloading everything  you should make another video

**Reply by FurkanGozukara** [reply]
> you kind of lost me in the video how do you translate after downloading everything you should make another video

for translation task you just need to give it as an input. however, translation works only for other languages to english.

**Comment by leonelcba** [comment]
Hi, I get this error when trying to install Whisper. Do you know why?

![Captura de pantalla 2022-11-04 121114](https://user-images.githubusercontent.com/117379507/200009540-7d6914cb-336b-4e62-b60d-21b2212cc801.png)

**Reply by FurkanGozukara** [reply]
> Hi, I get this error when trying to install Whisper. Do you know why?
> 
> ![Captura de pantalla 2022-11-04 121114](https://user-images.githubusercontent.com/117379507/200009540-7d6914cb-336b-4e62-b60d-21b2212cc801.png)

do not use the latest version python. you can try 3.9.9

**Reply by leonelcba** [reply]
Thank you so much! It worked :)
